app/Controllers/PrestitiController.php

```
@@ -2159,6 +2159,91 @@ public function exportCsv(Request $request, Response $response, mysqli $db): Res
+            $result = $service->sendManualRecall($loanId);
+            if ($result['success']) {
+                $sent++;
+            } else {
+                $skipped++;
+            }
+        }
+
+        return $response
+            ->withHeader('Location', $backUrl . '?bulk_recalled=' . $sent . '&bulk_recall_skipped=' . $skipped)
+            ->withStatus(302);
+    }
```

PHPMD (2.15.0)

[error] Avoid assigning values to variables in if clauses and the like (line '2193', column '13'). (undefined)

(IfStatementAssignment)